pandas/_libs/tslibs/offsets.pyx

```diff
@@ -2477,11 +2477,57 @@ cdef class MonthEnd(MonthOffset):
     """
     DateOffset of one month end.
 
+    Parameters
+    ----------
+    n : int, default 1
+        n = 0 -> offset to the end of month.
+        n = 1 -> depend on the given date:
+            a) Given date is on an anchor point (last day of the month) -> offset to the end of the following month.
+            b) Given date is not on an anchor point -> offset to the end of the same month.
+    
```

Comment:
does this look right? can you try building this page of the docs to check please? https://pandas.pydata.org/docs/dev/development/contributing_documentation.html

Reply:
Ok, but I will need more time for this. I still have trouble with setting up virtual environment (spend one hour with it today and will continue tomorrow)

Reply:
What is the version of Python you are using? I am created venv on top of Python 3.10, and trying to install package from requirements-dev.txt, but it always stuck at installation of boto3:
` 
  >>Using cached boto3-1.24.38-py3-none-any.whl (132 kB)
  >>Using cached boto3-1.24.37-py3-none-any.whl (132 kB)
  >>Using cached boto3-1.24.36-py3-none-any.whl (132 kB)
  Using cached boto3-1.24.35-py3-none-any.whl (132 kB)
  Using cached boto3-1.24.34-py3-none-any.whl (132 kB)
  Using cached boto3-1.24.33-py3-none-any.whl (132 kB)
  Using cached boto3-1.24.32-py3-none-any.whl (132 kB)`

Is it better to use older version of Python - e.g. 3.9? Thanks

Reply:
I'd suggest using mamba https://pandas.pydata.org/docs/dev/development/contributing_environment.html#option-1a-using-mamba-recommended

Reply:
please do ask (here or on Slack) if it's unclear / anything trips you up

Reply:
Thank you so much. I managed to install and setup mamba environment. However I have the same issue as I had before when I created virtual environment in PyCharm (missing module 'pandas._libs.interval'):
 
` "C:\Programming\Python\pandas\pandas\_libs\__init__.py", line 13, in <module>
    from pandas._libs.interval import Interval
ModuleNotFoundError: No module named 'pandas._libs.interval'`

Do  you know what could be causing this issue? Thanks

Reply:
yeah you'll need to rebuild your C extensions
```
python setup.py build_ext -j 4
```

Reply:
this needs doing any time one of the .pyx or .c files changes, which could happen if you've just fetched and merge upstream/main

Reply:
Thanks for a quick turnaround. However, when I run the commend you've suggested, I have another error:

`error: command 'cl.exe' failed: None`

I am doing my best to help, but I feel I am discovering more problems than solutions. Sorry!

Reply:
Will look at it again tomorrow. Hopefully I will make better progress 🤞

Reply:
no idea - could you show your complete traceback please?

Reply:
I tried to run it again, and also installed C++ Build Tools, but still receiving similar error. Please see below the full traceback:
[Traceback_error.txt](https://github.com/pandas-dev/pandas/files/9833587/Traceback_error.txt)

Reply:
I don't know, sorry - you could ask on [Slack ](https://pandas.pydata.org/docs/dev/development/community.html?highlight=slack#community-slack), or try using Ubuntu via the WSL on Windows

Reply:
Hi, thank you. I don't want to sound like giving up. However unfortunately, this is going deeper and deeper technical, and I feel I am not able to effectively contribute. Is it OK, if I share the updated text of documentation in thread under the issue, and leave it for someone more technically capable to proceed with implementation?

Reply:
sure, that's fine, someone else can take this forwards, and if you feel like trying again, feel free to reach out and ask

Reply:
Ok, thanks. Maybe I will give it another go this weekend. Will let you know how I decide to proceed then. Thanks